core/integration/features/spot/liquidity_provision/0034-PROB-008.feature

```diff
@@ -0,0 +1,118 @@
+      | lp3 | lp3   | BTC/ETH   | 2000              | 0.4 | submission |
+
+    And the parties should have the following account balances:
+      | party | asset | market id | general |
+      | lp1   | BTC   | BTC/ETH   | 600     |
+      | lp1   | ETH   | BTC/ETH   | 2000    |
+
+    Then the network moves ahead "1" blocks 
+    Then the market data for the market "BTC/ETH" should be:
+      | mark price | trading mode                 | auction trigger         | target stake | supplied stake | open interest |
+      | 0          | TRADING_MODE_OPENING_AUCTION | AUCTION_TRIGGER_OPENING | 4800         | 6000           | 0             |
+
+    # place orders and generate trades
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference    | only |
+      | party1 | BTC/ETH   | buy  | 1      | 14    | 0                | TYPE_LIMIT | TIF_GTC | party-order1 |      |
+      | party1 | BTC/ETH   | buy  | 1      | 15    | 0                | TYPE_LIMIT | TIF_GTC | party-order3 |      |
+      | party2 | BTC/ETH   | sell | 1      | 15    | 0                | TYPE_LIMIT | TIF_GTC | party-order4 |      |
+      | party2 | BTC/ETH   | sell | 1      | 18    | 0                | TYPE_LIMIT | TIF_GTC | party-order2 |      |
+
+    When the network moves ahead "1" blocks
+
+    Then the market data for the market "BTC/ETH" should be:
+      | mark price | trading mode            | auction trigger             | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 15         | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED | 3600000 | 4         | 52        | 4800         | 6000           | 0             |
+
+    And the parties place the following pegged orders:
+      | party | market id | side | volume | pegged reference | offset | reference |
+      | lp1   | BTC/ETH   | buy  | 10     | BID              | 7      | lp1-b     |
+      | lp1   | BTC/ETH   | sell | 10     | ASK              | 7      | lp1-s     |
+      | lp2   | BTC/ETH   | buy  | 10     | BID              | 2      | lp2-b     |
+      | lp2   | BTC/ETH   | sell | 10     | ASK              | 2      | lp2-s     |
+      | lp3   | BTC/ETH   | buy  | 10     | BID              | 1      | lp3-b     |
+      | lp3   | BTC/ETH   | sell | 10     | ASK              | 1      | lp3-s     |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference    | only |
+      | party1 | BTC/ETH   | buy  | 100    | 15    | 0                | TYPE_LIMIT | TIF_GTC | party-order3 |      |
+      | party2 | BTC/ETH   | sell | 100    | 15    | 1                | TYPE_LIMIT | TIF_GTC | party-order4 |      |
+
+    Then the accumulated liquidity fees should be "6000" for the market "BTC/ETH"
+    When the network moves ahead "4" blocks
+
```

Comment:
A comment on how this is testing that the prob. of. trading is decreasing would be good. I mean I see it - the LPs with bigger offset get lower rewards but it's a bit cryptic without an explanation, given the wording of the AC.